logstash-core/lib/logstash/config/source/local.rb

```diff
@@ -102,7 +102,7 @@ def get_unmatched_files
       end
 
       def valid_encoding?(content)
-        content.ascii_only? && content.valid_encoding?
+        content.ascii_only? || content.valid_encoding?
```

Comment:
[valid_encoding](https://github.com/jruby/jruby/blob/dd0b88e44efe91ed1cf163b99abac0362d614d1b/core/src/main/java/org/jruby/RubyString.java#L5435-L5438) just seems to test if it's not 16 or 32 bit encoded: https://github.com/jruby/jruby/blob/c24f7fa865d96c76235ebab591db938b0c52851b/core/src/main/java/org/jruby/util/StringSupport.java#L62-L69, so I'm guessing it's ok to have only that check